File: app/src/main/java/com/battlelancer/seriesguide/shows/episodes/EpisodeFlags.java
// Copyright 2023 Uwe Trottmann
// SPDX-License-Identifier: Apache-2.0

package com.battlelancer.seriesguide.shows.episodes;

/**
 * {@link #WATCHED}, {@link #SKIPPED} and {@link #UNWATCHED} flag for episode watched state.
 */
public interface EpisodeFlags {

    int UNWATCHED = 0x0;

    int WATCHED = 0x1;

    int SKIPPED = 0x2;

}

File: app/src/main/java/com/battlelancer/seriesguide/shows/episodes/EpisodeFlags.kt
// SPDX-License-Identifier: Apache-2.0
// Copyright 2013-2024 Uwe Trottmann

package com.battlelancer.seriesguide.shows.episodes

/**
 * [WATCHED], [SKIPPED] and [UNWATCHED] flag for episode watched state.
 */
object EpisodeFlags {
    const val UNWATCHED: Int = 0x0

    const val WATCHED: Int = 0x1

    const val SKIPPED: Int = 0x2
}
